resources/js/components/ActionButtonField.vue

```diff
@@ -33,6 +34,7 @@
     // Computed
     const text = computed(() => props?.field?.text || null);
     const icon = computed(() => props?.field?.icon || null);
+    const iconIsUrl = computed(() => icon?.value?.startsWith('http') === true);
```

Comment:
Maybe it better to use separate method in ActionButton.php to set `imageUrlOfIcon` (or smth like this)?
I think it will be cleaner for users who wants to set icon as svg or as image from url